lib/chef/knife/mixin/helper.rb

```diff
@@ -27,6 +27,7 @@ def set_mode(mode)
       def merge_values(json, file)
         values = {}
         values.merge!(values_from_file(file)) if file
+        validate_json(json)
```

Comment:
Should this validation be moved into `values_from_json`, so that bad values in json files will get caught in `values_from_file`?

Reply:
yes, will move it, thanks